### ISSUE TITLE
When downloading new versions, how can I download only new pictures from list?

### ISSUE THREAD

**Issue by esotericer1** [Nandaka/PixivUtil2#454]
Hi, thank you always for your wonderful program. I have a question : after I install the latest versions downloader, is it possible to only download new images from pixiv(using 'download by list'), that I haven't downloaded from my previous versions? 
So far, when I install the latest versions I had to run the 'download by list' all over again to get new images from listed artists that I haven't downloaded previously. But the problem with this is that it also re-downloads images that I've deleted. For example, even though I deleted a pic from an artist's folder, when I install the latest version and download's that artist's gallery(using list) the deleted file is back all over again. Also, if artists have changed their username, the new versions download their galleries all over again, leading to redundant files. 

To conclude, is it possible to incorporate a 'database file', or 'download log' of sorts from previous downloader into new downloader? So as to prevent repetition of downloads.

**Comment by NHOrus** [comment]
That's called db.sqlite and it contains everything you want. Copy it over or extract new version over old one25 дек. 2018 г. 22:40 пользователь esotericer1 <notifications@github.com> написал:Hi, thank you always for your wonderful program. I have a question : after I install the latest versions downloader, is it possible to only download new images from pixiv(using 'download by list'), that I haven't downloaded from my previous versions?
So far, when I install the latest versions I had to run the 'download by list' all over again to get new images from listed artists that I haven't downloaded previously. But the problem with this is that it also re-downloads images that I've deleted. For example, even though I deleted a pic from an artist's folder, when I install the latest version and download's that artist's gallery(using list) the deleted file is back all over again. Also, if artists have changed their username, the new versions download their galleries all over again, leading to redundant files.
To conclude, is it possible to incorporate a 'database file', or 'download log' of sorts from previous downloader into new downloader? So as to prevent repetition of downloads.
